openy.install

```diff
@@ -344,3 +344,14 @@ function openy_update_8026() {
   $config_importer = \Drupal::service('openy_upgrade_tool.param_updater');
   $config_importer->update($config, 'editor.editor.full_html', 'settings.toolbar');
 }
+
+/**
+ * Enable GroupEx Pro related modules.
+ */
+function openy_update_8027() {
+  // Enable module.
+  \Drupal::service('module_installer')->install([
```

Comment:
let's foreach loop install one-by-one not all at once. Saving memory is a must have here